faker/providers/person/zh_TW/__init__.py

```diff
@@ -1,75 +1,431 @@
-                  "戚", "謝", "鄒", "喻", "柏", "水", "竇", "章",
-                  "雲", "蘇", "潘", "葛", "奚", "範", "彭", "郎",
-                  "魯", "韋", "昌", "馬", "苗", "鳳", "花", "方",
-                  "俞", "任", "袁", "柳", "酆", "鮑", "史", "唐",
-                  "費", "廉", "岑", "薛", "雷", "賀", "倪", "湯",
-                  "滕", "殷", "羅", "畢", "郝", "鄔", "安", "常",
-                  "樂", "於", "時", "傅", "皮", "卞", "齊", "康",
-                  "伍", "餘", "元", "蔔", "顧", "孟", "平", "黃",
-                  "和", "穆", "蕭", "尹", "姚", "邵", "湛", "汪",
-                  "祁", "毛", "禹", "狄", "米", "貝", "明", "臧",
-                  "計", "伏", "成", "戴", "談", "宋", "茅", "龐",
-                  "熊", "紀", "舒", "屈", "項", "祝", "董", "梁",
-                  "杜", "阮", "藍", "閔", "席", "季", "麻", "強",
-                  "賈", "路", "婁", "危", "江", "童", "顏", "郭",
-                  "梅", "盛", "林", "刁", "鍾", "徐", "邱", "駱",
-                  "高", "夏", "蔡", "田", "樊", "胡", "淩", "霍",
-                  "虞", "萬", "支", "柯", "昝", "管", "盧", "莫",
-                  "柯", "房", "裘", "繆", "幹", "解", "應", "宗",
-                  "丁", "宣", "賁", "鄧", "鬱", "單", "杭", "洪",
-                  "包", "諸", "左", "石", "崔", "吉", "鈕", "龔",
-                  "程", "嵇", "邢", "滑", "裴", "陸", "榮", "翁",
-                  "荀", "羊", "于", "惠", "甄", "曲", "封",
-                  "芮", "羿", "儲", "靳", "汲", "邴", "糜", "松",
-                  "井", "段", "富", "巫", "烏", "焦", "巴", "弓",
-                  "牧", "隗", "山", "穀", "車", "侯", "宓", "蓬",
-                  "全", "郗", "班", "仰", "秋", "仲", "伊", "宮",
-                  "甯", "欒", "暴", "甘", "鈄", "曆", "戎",
-                  "祖", "武", "符", "劉", "景", "詹", "束", "龍",
-                  "葉", "幸", "司", "韶", "郜", "黎", "薊", "溥",
-                  "印", "宿", "白", "懷", "蒲", "邰", "從", "鄂",
-                  "索", "鹹", "籍", "賴", "卓", "藺", "蒙",
-                  "池", "喬", "陽", "鬱", "胥", "能", "蒼", "雙",
-                  "聞", "莘", "党", "翟", "譚", "貢", "勞", "逄",
-                  "姬", "申", "扶", "堵", "冉", "宰", "酈", "雍",
-                  "卻", "璩", "桑", "桂", "濮", "牛", "壽", "通",
-                  "邊", "扈", "燕", "冀", "浦", "尚", "農",
-                  "溫", "別", "莊", "晏", "柴", "瞿", "閻", "充",
-                  "慕", "連", "茹", "習", "宦", "艾", "魚", "容",
-                  "向", "古", "易", "慎", "戈", "廖", "庾", "終",
-                  "暨", "居", "衡", "步", "都", "耿", "滿", "弘",
-                  "匡", "國", "文", "寇", "廣", "祿", "闕", "東",
-                  "歐", "殳", "沃", "利", "蔚", "越", "夔", "隆",
-                  "師", "鞏", "厙", "聶", "晁", "勾", "敖", "融",
-                  "冷", "訾", "辛", "闞", "那", "簡", "饒",
-                  "曾", "毋", "沙", "養", "鞠", "須", "豐",
-                  "巢", "關", "蒯", "相", "查", "後", "荊", "紅",
-                  "游", "竺", "權", "逮", "盍", "益", "桓", "公",
-                  "司馬", "上官", "歐陽", "東方", "公羊",
-                  "公冶", "淳于", "公孫", "慕容", "司徒", "司空")
+    # From https://zh.wikipedia.org/wiki/%E4%B8%AD%E5%9B%BD%E5%A7%93%E6%B0%8F%E6%8E%92%E5%90%8D
+    last_names = OrderedDict((
```

Comment:
This appear to be the same list as https://github.com/joke2k/faker/blob/master/faker/providers/person/zh_CN/__init__.py#L29. You should use data specific to Taiwan for the Taiwanese provider.

Reply:
The data source is same, because the data source contains the last name data of all Chinese regions (China Mainland, Taiwan, Hong Kong and Macao ).
I just translated Simplified Chinese into Traditional Chinese